doc/hardware/sensors.rst

```diff
@@ -0,0 +1,41 @@
+
+Sensors list
+##################################
+
+.. table:: Supported sensors on MinnowBoard MAX, Galileo GEN 2 and Edison boards
+
+    ============== ============================================= ==========================================================================
+    Sensor Name     Sensor Type                                    Comment
+    ============== ============================================= ==========================================================================
+    BMA180          Accel                                          missing driver on Edison
+    BMA250          Accel                                          missing driver on Edison
+    LIS3DH          Accel                                          missing driver on Edison
+    MMA7455         Accel                                          missing driver on Edison
+    BMA255          Accel                                          missing driver on Edison
+    LIS2DH12TR      Accel                                          missing driver on Edison
+    BMA222E         Accel                                          missing driver on Edison
+    LSM330DLC       Gyro
+    MPU6050         Gyro
+    MPU9250         Gyro
+    L3G4200         Gyro
+    LSM303          Magnet
```

Comment:
agreed with all @dbkinder's comments but the column-per-board, as it won't scale when we support more boards :-)

I'd say ideally we create one doc/hardware/XXX.rst, like doc/hardware/lsm303.rst, inside that you can point to adafruit and other places to buy, then also provide a sample usage with FBP and/or JS.